src/components/GettingThere.astro

```diff
@@ -213,21 +213,24 @@ import Map from "../img/Map.png";
                 <p>
                     The Matrix Conference ticket holders are eligible to use the "<a
                     href="https://www.veranstaltungsticket-bahn.de/?event=35482&language=en"
-                    target="_blank">Event Ticket</a>" special offer by Deutsche Bahn to and from the conference!
+                    target="_blank">Event Ticket</a>" special offer by Deutsche Bahn to and from
+                    the conference!
                 </p>
             </div>
         </div>
         <div class="item">
             <h3>By Plane</h3>
             <p>
                 Strasbourg's Airport (SXB) is another option for attendees
-                coming from farther. The <span class="ter">TER</span> train connects
-                the Airport to Strasbourg's city center in about 30 min. A taxi ride
-                will get you there in bout 20 min.
+                coming from farther. The <span class="ter">TER</span> train from <a
+                href="https://www.sncf-connect.com/en-en/home/search/od?destinationLabel=Strasbourg+Airport&destinationId=RESARAIL_STA_8721452"
```

Comment:
This link sets the Airport as destination instead of startpoint

Reply:
(Yes, they just process natural language directions. Awful concept.)

Reply:
When you click "Buy your train ticket", the big, bold title of the following page is a date selection where you can select for which date you wanna buy your ticket. I mean, seriously, we can't completely take away the user's need to use their own brain.

<img width="1306" height="232" alt="image" src="https://github.com/user-attachments/assets/0b7cedee-7952-4d59-a81e-f8209a1c8889" />